http/1.5/modules/ROOT/nav.adoc

```diff
@@ -1,24 +1,25 @@
 .xref:index.adoc[HTTP Connector]
 * xref:index.adoc[About HTTP Connector]
-* xref:http-about-http-request.adoc[HTTP Requests and Responses]
-* xref:http-authentication.adoc[Authenticate HTTP Requests]
+** xref:http-listener-ref.adoc[HTTP Listener Configuration Reference]
+** xref:http-request-ref.adoc[HTTP Request Configuration Reference]
+** xref:http-about-http-request.adoc[HTTP Requests and Responses]
+** xref:http-authentication.adoc[Authenticate HTTP Requests]
+** xref:http-about-loading-static-resource.adoc[About Loading a Static Resource]
```

Comment:
I'll take a look when I get to the text, but seeing only this title, I wonder what the focus of "About Loading" is: how to load? why to load? best way to load? when to load?